factory/config.go

```diff
@@ -47,12 +47,12 @@ type Configuration struct {
 }
 
 type Sbi struct {
+	Tls          *Tls   `yaml:"tls,omitempty"`
 	Scheme       string `yaml:"scheme"`
 	RegisterIPv4 string `yaml:"registerIPv4,omitempty"` // IP that is registered at NRF.
 	// IPv6Addr string `yaml:"ipv6Addr,omitempty"`
```

Comment:
@thakurajayL, should we remove this line (and other commented out code that is related to IPv6? Or are we planning to enable IPv6 support in the future?

Reply:
We can remove it. We can bring it back when we wish to enable.

Reply:
@patriciareinoso, let's remove stuff related to IPv6 that is commented out, ok? We can do it in a another PR